packages/components/src/internal/components/user/UserDetailHeader.spec.tsx

```diff
@@ -10,40 +10,21 @@ import { UserDetailHeader } from './UserDetailHeader';
 
 describe('<UserDetailHeader/>', () => {
     test('default properties', () => {
-        const component = (
-            <UserDetailHeader
-                title="Title"
-                user={TEST_USER_READER}
-                userProperties={fromJS({})}
-                dateFormat={undefined}
-            />
-        );
+        const component = <UserDetailHeader dateFormat="YYYY-MM-DD" title="Title" user={TEST_USER_READER} />;
```

Comment:
Yes. The `dateFormat` property was marked as required by the interface. I've marked it as optional on this and the underlying `getUserLastLogin` since it is apparently intended to support it.